Ardupilog.m

```diff
@@ -132,6 +133,9 @@ function delete(obj)
                     continue;
                 end
 
+                msgLen = obj.fmt_cell{i,3};
+                data = obj.isolateMsgData(msgId,msgLen,allHeaderCandidates);
+
```

Comment:
Okay, I think you do that for all the messages, so as to have all the locations of the valid message headers. Understandable, but it will cause a major hit in efficiency. Effectively, it will be the same as parsing ALL the headers.

We should evaluate if this delay is unacceptable or not.